docs/pages/blog/tenerife-retreat.md

```diff
@@ -0,0 +1,162 @@
+---
+title: MUI's company retreat in Tenerife: a recap
+description: Our internationally distributed startup gathered on a remote island to get to know each other better. Here's what happened!
+date: 2022-07-25T00:00:00.000Z
+authors: ['samuelsycamore']
+card: false
+tags: ['Company']
+---
+
+One of the toughest challenges to overcome as a fully remote team is fostering a supportive and inclusive company culture.
+How do you build trust, comradery, and—dare I say it?—_friendships_, when there are so few opportunities for spontaneous social interactions in our daily work routines?
+
+At MUI, we've chosen to prioritize in-person company retreats as a way to grow together and cultivate the culture that informs how we operate.
+
+![Photo of the MUI team in front of the pool at our accommodations in Tenerife](/static/blog/tenerife-retreat/group-photo.jpeg)
```

Comment:
The picture is dark, it's kind of depressing  🙃. If we can't get @mbrookes' drone picture in time, maybe we could explore color manipulations, e.g. with Adobe Lightroom.

Reply:
The new one is much better :D